public/v1/daoyuan/js/couponEdit.js

```diff
@@ -54,18 +54,19 @@ $(document).ready(function(){
      rebateRule: rebateRule,
      valid: $('#update-valid').val()
     };
-    
     console.log(data);
     $.ajax({
       type: "PUT",
       url: putUrl,
-      data: data,
+      data: JSON.stringify(data),
+      contentType: "application/json",
       beforeSend: function(xhr)
       {
         xhr.setRequestHeader('x-access-token', getCookieByName('x-access-token'));
       },
       success: function(data, textStatus, xhr)
-      { 
+      {
+        debugger;
```

Comment:
that is cool... work fine...I think I have done 'data: JSON.stringify(data)', but it still did not work yesterday...magic